content/swiftui/concepts/views/terms/table/table.md

```diff
@@ -0,0 +1,60 @@
+---
+Title: `Table`
+Description: `A container that presents rows of data arranged in one or more columns, optionally providing the ability to select one or more members.`
+Subjects: Mobile Development, iOS application, SwiftUI
+Tags: SwiftUI, iOS, table, view
+
+---
+
+
+A **Table** is a container that presents rows of data arranged in one or more columns, optionally providing the ability to select one or more members.
+
+Tables are commonly created from collections of data.
+If there are more rows than can fit in the available space, the table provides vertical scrolling automatically. On macOS, the table also provides horizontal scrolling if there are more columns than can fit in the width of the view.
+
+>struct Product: Identifiable {
+ let id: UUID()
+ let name: String
+ let color: String
+ let price: Double
+}
+
+>@State private var products = [
+	Product(name: "Pen", color: "Black", price: 2.99),
+	Product(name: "Eraser", color: "White", price: 1.49),
+	Product(name: Pencil, color: "Green", price: 0.99)
+	]
+
+>struct ProductTable: View {
+	var body: some View {
+		Table (products) {
+			TableColumn ("Product Name", value: \.name)
+			TableColumn("Product Color", value: \.color)
+			TableColumn("Product Price", value: \.price) 
+		}
+	 }
+	}
```

Comment:
```suggestion
struct Product: Identifiable {
    let id: UUID()
    let name: String
    let color: String
    let price: Double
}

@State private var products = [
    Product(name: "Pen", color: "Black", price: 2.99),
    Product(name: "Eraser", color: "White", price: 1.49),
    Product(name: Pencil, color: "Green", price: 0.99)
]

struct ProductTable: View {
    var body: some View {
        Table (products) {
            TableColumn ("Product Name", value: \.name)
            TableColumn("Product Color", value: \.color)
            TableColumn("Product Price", value: \.price) 
        }
    }
}
```

Reply:
and like @SSwiniarski mentioned, place this in a code block. To open the block use three backticks and swift: "```swift" then close with three backticks

Reply:
This code block would also need some explanation. If you want to simplify the example in terms of complexity, feel free to do so and we can reevaluate